samples/http-client-examples/http-client-httpbin.spec.ts

```diff
@@ -0,0 +1,89 @@
+import {
+  executeHttpRequest,
+  executeHttpRequestWithOrigin,
+  ParameterEncoder
+} from '@sap-cloud-sdk/http-client';
+
+/* 
+
+This spec file includes usage examples for the HTTP client that make use of the httpbin.org service.
+The examples in this file should be exactly the same as on https://sap.github.io/cloud-sdk/docs/js/features/connectivity/generic-http-client
+
+*/
+
+describe('HTTP Client Usage Examples using httpbin.org', () => {
+  it('Show simple HTTP Post without fetchCsrfToken', async () => {
+    const response = await executeHttpRequest(
+      {
+        url: `https://httpbin.org/post`
```

Comment:
[q] is httpbin stable or is a mocked URL the best?

Reply:
What would a "mocked url" in this case be?

I do have the other spec file which uses our local express-http-server, but I wanted to have tests that are 1:1 the same as on the docs page and don't require a local server.

Sure I can't guarantee how long httpbin will exist, but I know it for years and think it is a quite reliable page for that purpose.

Reply:
discussed in person, that it is ok to use it so the code snippets are also directly usable as copy paste.